index.bs

```diff
@@ -993,6 +860,8 @@ a [=trigger state=] |triggerState|:
     |triggerState|'s [=trigger state/report window=].
 1. Return |fakeReport|.
 
+Issue: Check for transient activation.
```

Comment:
Does this belong here in obtaining a fake report? I think we want an issue to monkeypatch https://html.spec.whatwg.org/multipage/links.html#following-hyperlinks-2 and check for activation there prior to invoking navigation (unless we kept navigation monkeypatches and checked for activation in one place)

Reply:
Relocated the TODO for now